### ISSUE TITLE
TestNiftyExamples crashes

### ISSUE THREAD

**Issue by Ali-RS** [jMonkeyEngine/jmonkeyengine#1188]
Both Textfield and Drop down list examples throw this error: 

```
java.lang.NoSuchMethodError: de.lessvoid.nifty.controls.label.builder.LabelBuilder.text(Ljava/lang/String;)V
	at de.lessvoid.nifty.examples.textfield.TextFieldDemoStartScreen.bind(TextFieldDemoStartScreen.java:44)
	at de.lessvoid.nifty.screen.Screen.startScreen(Screen.java:225)
	at de.lessvoid.nifty.Nifty.gotoScreenInternal(Nifty.java:767)
	at de.lessvoid.nifty.Nifty.access$400(Nifty.java:87)
	at de.lessvoid.nifty.Nifty$1.perform(Nifty.java:724)
	at de.lessvoid.nifty.elements.EndOfFrameElementAction.perform(EndOfFrameElementAction.java:24)
	at de.lessvoid.nifty.Nifty.executeEndOfFrameElementActionsInternal(Nifty.java:490)
	at de.lessvoid.nifty.Nifty.handleDynamicElements(Nifty.java:442)
	at de.lessvoid.nifty.Nifty.update(Nifty.java:373)
	at com.jme3.niftygui.InputSystemJme.endInput(InputSystemJme.java:112)
	at com.jme3.input.InputManager.processQueue(InputManager.java:853)
	at com.jme3.input.InputManager.update(InputManager.java:917)
	at com.jme3.app.LegacyApplication.update(LegacyApplication.java:724)
	at com.jme3.app.SimpleApplication.update(SimpleApplication.java:244)
	at com.jme3.system.lwjgl.LwjglWindow.runLoop(LwjglWindow.java:503)
	at com.jme3.system.lwjgl.LwjglWindow.run(LwjglWindow.java:585)
	at com.jme3.system.lwjgl.LwjglWindow.create(LwjglWindow.java:427)
	at com.jme3.app.LegacyApplication.start(LegacyApplication.java:463)
	at com.jme3.app.LegacyApplication.start(LegacyApplication.java:424)
	at com.jme3.app.SimpleApplication.start(SimpleApplication.java:125)
	at jme3test.niftygui.TestNiftyExamples.main(TestNiftyExamples.java:46)
```

Tested with the master branch and Java 12

**Comment by stephengold** [comment]
It's a build issue. `jme3-testdata` references v1.4.1 of `nifty-examples`, whereas `jme3-niftygui` references v1.4.3 of `nifty` and `nifty-default-controls`.

**Comment by stephengold** [comment]
Fixed in `master` branch at dcb66977ba903ad4ac5eda43c70cf2b8f5db5843 .
